pkg/sql/opt/operator.go

```diff
@@ -396,7 +397,7 @@ func AggregatesCanMerge(inner, outer Operator) bool {
 
 	case AnyNotNullAggOp, BitAndAggOp, BitOrAggOp, BoolAndOp,
 		BoolOrOp, ConstAggOp, ConstNotNullAggOp, FirstAggOp,
-		MaxOp, MinOp, STMakeLineOp, STExtentOp, STUnionOp, SumOp, SumIntOp, XorAggOp:
+		MaxOp, MinOp, STMakeLineOp, STCollectOp, STExtentOp, STUnionOp, SumOp, SumIntOp, XorAggOp:
```

Comment:
i think it shouldn't be able to merge because the ordering of st_collect can change.

Reply:
Good point, thanks - updated.